prow/tide/tide.go

```diff
@@ -146,14 +146,14 @@ func (c *Controller) setStatuses(all, pool []PullRequest) error {
 				actualDesc = string(ctx.Description)
 			}
 		}
-		if wantState != actualState || wantDesc != actualDesc {
+		if wantState != strings.ToLower(string(actualState)) || wantDesc != actualDesc {
```

Comment:
Why `strings.ToLower`?

Reply:
`wantState` is a v3 (all lower case) value while `actualState` is a v4 (all caps) value.

Reply:
Ohh, I see.